README.md

```diff
@@ -3,37 +3,188 @@
 [![PyPI wheel](https://img.shields.io/pypi/wheel/econml.svg)](https://pypi.org/project/econml/)
 [![Supported Python versions](https://img.shields.io/pypi/pyversions/econml.svg)](https://pypi.org/project/econml/)
 
-# Introduction 
-
-The [ALICE project](https://www.microsoft.com/en-us/research/project/alice/) at Microsoft Research is 
-aimed at applying Artificial Intelligence concepts to economic decision making.  The Microsoft EconML 
-package is part of that project, providing a toolkit that combines state-of-the-art machine learning 
-techniques with econometrics in order to bring automation to complex causal inference problems.  This 
-toolkit is designed to measure the causal effect of some treatment variable(s) `t` on an outcome 
-variable `y`, controlling for a set of features `x`.  For more information about how to use this package, 
+
+
+<h1><img src="https://www.microsoft.com/en-us/research/wp-content/uploads/2016/12/MSR-ALICE-HeaderGraphic-1920x720_1-800x550.jpg" width="130px" align="left" style="margin-right: 10px;"> EconML: A Python Package for ML-Based Heterogeneous Treatment Effects Estimation</h1>
+
+**EconML** is a Python package for estimating heterogeneous treatment effects from observational data via machine learning. This package was designed and build as part of the [ALICE project](https://www.microsoft.com/en-us/research/project/alice/) at Microsoft Research with the goal to combine state-of-the-art machine learning 
+techniques with econometrics in order to bring automation to complex causal inference problems. The promise of EconML:
+
+* Implement recent techniques in the literature at the intersection of econometrics and machine learning
+* Maintain flexibility in modeling the effect heterogeneity (via techniques such as random forests, boosting, lasso and neural nets), while preserving the causal interpretation of the learned model and often offering valid confidence intervals
+* Unified API
```

Comment:
For parallelism, I think it would be better to phrase the last two bullets as "Use a unified API" and "Build on standard ..." so that they all start with verbs in the same tense.

Reply:
Done!